api/handlers/blockinfo_handler.go

```diff
@@ -790,6 +790,30 @@ func (h *BlockHandler) GetTxs() gin.HandlerFunc {
 	}
 }
 
+//GetDebtByHash handler for get debt by hash
+func (h *BlockHandler) GetDebtByHash() gin.HandlerFunc {
+	return func(c *gin.Context) {
+		dbClinet := h.DBClient
```

Comment:
dbClinet should be dbClient

Reply:
Previous version dbClinet unity, all files have been replaced with dbClient